index.ts

```diff
@@ -95,6 +95,15 @@ export declare namespace ens {
   function nameByHash(hash: string): string | null
 }
 
+export declare namespace arweave {
+  /// Returns `null` if the transaction is not found.
+  function transactionData(txId: string): Bytes | null;
+}
+export declare namespace box {
+  /// Returns `null` if the profile is not found.
```

Comment:
Small nit: `///` is not a thing in TypeScript/AssemblyScript land. I would use
```
/**
  * ...
  */
```
or
```
/** ... */
```
like we've done with most of the other documentation comments in this file. We _might_ want to generate API docs automatically at some point and then that'll likely help.